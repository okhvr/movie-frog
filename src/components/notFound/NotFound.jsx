import React, { Component } from 'react';

import './style.scss';

export default class NotFound extends Component {

  render() {
    return (
        <h4 className="notFound">Page not found :(</h4>
    );
  }
}
