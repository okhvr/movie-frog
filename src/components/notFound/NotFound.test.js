import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

describe('Footer component', () => {
  it('should render correctly', () => {
    const component = renderer.create(
        <NotFound/>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});