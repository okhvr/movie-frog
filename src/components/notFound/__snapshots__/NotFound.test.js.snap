// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<h4
  className="notFound"
>
  Page not found :(
</h4>
`;
